public/settings.dist.json

```
@@ -0,0 +1,8 @@
+{
+  "favicon": "/favicon.png",
+  "title": "mock_title",
+  "theme": { 
+    "logo": "/images/mocked-logo.png",
+    "style": []
+  }
+}
```

Please keep easy to replace placeholders here:

```suggestion
{
  "favicon": "<FAVICON>",
  "title": "<TITLE>",
  "theme": { 
    "logo": "<LOGO_PATH>",
    "style": [<STYLE_PATH>]
  }
}
```

---

done